modules/ROOT/pages/sending-data-from-arm-to-external-analytics-software.adoc

```diff
@@ -158,321 +102,259 @@ INDEXED_EXTRACTIONS = JSON
+. Under *Event Tracking*, enable *Splunk*.
+. Select the type of information to send from the *Level* menu. 
++
+See <<Logging Level Options>>.
+. Click the gear icon to specify your Splunk configuration:
++
+.The screenshot shows (*1*) the protocol option, (*2*) the data input types, and (*3*) the *Show more options* switch in the Splunk configuration window.
+image::hybrid-export-splunk-configure.png[Protocol option, data input types, and Show more options switch in the Splunk configuration window]
++
+See <<Splunk Configuration Options>>.
+. Specify the host (IP address or hostname) and management port of the server where Splunk is running. 
++
+* To send data to Splunk using the REST API, select *Rest API* and enter the username and password for your Splunk account.
+* To send data to Splunk over TCP, select *tcp* from the menu and enter the username and password for your Splunk account.
+* To send data to Splunk using the HTTP Event Collector, select *HTTP Event Collector* and paste the token from Splunk in the *Token* field:
++
+.The screenshot shows (*1*) the *HTTP Event Collector* option and (*2*) the *Token* field in the Splunk configuration window.
+image::hybrid-export-splunk-configure-http.png[HTTP Event Collector option and Token field in the Splunk configuration window]
 
-|`sslSecurityProtocol`
-|String
-|SSL Security Protocol to use in the https connection. Possible values: TLSv1_2, TLSv1_1, TLSv1, SSLv3.
-|Optional
-|TLSv1_2
 
-|`splunkIndexName`
-|String
-|Splunk index name where all the events must be sent. If the user has the rights,
-and the index doesn't exist, then the internal handler will create it.
-|Optional
-|main
+. If you want, click *Show more options* to specify formatting properties for the sent data:
++
+.The arrow shows the expanded options in the Splunk configuration window.
+image::hybrid-export-splunk-configure-advanced.png[Expanded options in the Splunk configuration window]
++
+See <<Splunk Configuration Options>>.
++
+The *SSL Protocol* option applies only to the REST API data input type.
++
+[IMPORTANT]
+If you set values for the Splunk index, Splunk source, and Splunk source type when you registered your data input in Splunk, any values you specify in this window overwrites those values.
 
-|`splunkSource`
-|String
-|The source used on the events sent to Splunk.
-|Optional
-|mule
+. Click *Apply* to save your changes.
 
-|`splunkSourceType`
-|String
-|The sourcetype used on the events sent to Splunk.
-|Optional
-|mule
+=== Splunk Configuration Options
 
-|`dateFormatPattern`
-|String
-|Date format used to format the timestamp.
-|Optional
-|yyyy-MM-dd'T'HH:mm:ssSZ
+Splunk integration includes the following configuration options:
 
+[cols="20,30,20"]
 |===
+|Field|Description|Default Value
 
-.Splunk Internal Handler Minimum Configuration Example
-[source,yaml]
-....
----
-  mule.agent.gw.http.handler.splunk:
-    host: 192.168.61.131
-    user: admin
-    pass: test
-....
-
-=== Rest API
+|Username
```

Comment:
Is this field name init. capped but the others lowercase in the UI?

Reply:
Yes. It's a cluster.